hydra/app/factories/bulkrax/acda_factory.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Bulkrax
+  # Used to create Acda records when performing Bulkrax imports.
+  # Based on Bulkrax's own Bulkrax::ObjectFactory.
+  class AcdaFactory
+    attr_reader :attributes, :object, :source_identifier_value,
+                :klass, :replace_files, :update_files, :work_identifier,
+                :related_parents_parsed_mapping, :importer_run_id
+
+    def initialize(attributes:, source_identifier_value:, work_identifier:, related_parents_parsed_mapping: nil, replace_files: false, user: nil, klass: nil, importer_run_id: nil, update_files: false)
+      @attributes = ActiveSupport::HashWithIndifferentAccess.new(attributes)
+      @user = user || User.batch_user
```

Comment:
No `batch_user` method currently exists on the `User` model; we will likely need to implement one